substrate/frame/revive/fixtures/contracts/drain.rs

```diff
@@ -36,6 +36,15 @@ pub extern "C" fn call() {
 
 	// Try to self-destruct by sending more balance to the 0 address.
 	// The call will fail because a contract transfer has a keep alive requirement.
-	let res = api::transfer(&[0u8; 20], &u256_bytes(balance));
```

Comment:
can't we keep this as an alias to call?

Reply:
I prefer the uapi to just thinly expose the syscalls. Otherwise the line gets blurry quickly. For example should we then also have an alias for a static call? Or for depositing events with no topics, a revert alias to return etc. Consumers of this API can still alias it themself if they want.